fay.cabal

```diff
@@ -133,7 +133,6 @@ library
     Language.Haskell.Names.SyntaxUtils
     Language.Haskell.Names.Types
     Paths_fay
-
   build-depends:
       base >= 4.5 && < 4.11
```

Comment:
The base lower bound needs to be changed since we are using some newer GHC features, I guess the easiest way would be to just try compiling with older compilers to figure out which versions we do support.